doc/develop/west/manifest.rst

```diff
@@ -980,6 +983,45 @@ is inactive.
 The final group filter specified by :file:`parent/west.yml` and the
 ``manifest.group-filter`` configuration option is ``[+unstable,-optional]``.
 
+
+Filtering projects using a custom manifest
+==========================================
```

Comment:
There are already several examples of how to do this on this page. Read the sections starting at `west-manifest-ex3.1`. Why do you need this? Why isn't it following the standard format for examples on this page?